src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,12 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
-}
\ No newline at end of file
+    } 
+    
+    public static String[] trim(String []str){
+        for(int i=0;i<str.length;i++){
```

Comment:
There is coding standard that we must follow here. Please refer to https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true
i.e. adding a space between operands and operators, a space after `for` keyword and a space after every `;` and between `)` and `{`.